validation/src/test/scala/hmda/validation/dsl/PredicateRegExSpec.scala

```diff
@@ -0,0 +1,109 @@
+package hmda.validation.dsl
+
+import hmda.parser.fi.ts.TsGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+import hmda.validation.dsl.PredicateRegEx._
+import hmda.validation.dsl.PredicateSyntax._
+import org.scalacheck.Gen
+
+class PredicateRegExSpec
+    extends PropSpec
+    with PropertyChecks
+    with TsGenerators
+    with MustMatchers {
+
+  // Email **************
+
+  property("A valid email must pass the email regex") {
```

Comment:
Since the `emailGen` generator only generates a small subset of valid emails, I'd like to see another spec that shows specific examples, with things like numerals, symbols, etc. A few I thought of that pass:

``` 
    val testCases = List("what.e.v.e.r@example.co.uk", "6237468@example.gov",
      "f@a.ke", "lol@12345.lol", "hi__mom@example.club")
```